org.kde.Kid3.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: org.kde.Kid3
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: kid3
+
+rename-desktop-file: org.kde.kid3.desktop
+rename-appdata-file: org.kde.kid3.appdata.xml
+rename-icon: kid3-qt
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
```

Comment:
```suggestion
```
you're using fallback-x11 already

Reply:
They can co-exist for compatibility with older flatpak versions.

Reply:
I know, but it was added in `0.11.3` and per https://ahayzen.com/direct/flathub.html#downloadsbyflatpak almost every download that hits Flathub is using a newer version of flatpak.

Reply:
Removed it.